chapter_deep-learning-computation/parameters_vn.md

```diff
@@ -339,7 +345,11 @@ In the example below, we  pick a decidedly bizarre and nontrivial distribution,
 We draw the coefficients from the following distribution:
 -->
 
-*dịch đoạn phía trên*
+Đôi khi, các phương thức khởi tạo mà ta cần không có sẵn trong mô-đun `init`.
+Trong trường hợp đó, ta có thể lập trình một lớp con của lớp `Initializer` và sử dụng nó như bất kỳ phương thức khởi tạo nào khác.
+Thông thường, ta chỉ cần lập trình hàm `_init_weight` để thay đổi tham số `ndarray` đầu vào (`data`) bằng giá trị khởi tạo mong muốn.
+Trong ví dụ bên dưới, ta sẽ chọn một phân phối kỳ lạ và không tầm thường để chứng minh luận điểm trên.
```

Comment:
```suggestion
Trong ví dụ bên dưới, ta sẽ chọn một phân phối kỳ lạ và bất thường để chứng minh luận điểm trên.
```
Mình chưa nghĩ từ `bất thường` là hoàn toàn phù hợp. Nên chỗ này có thể sẽ cần thảo luận thêm

Reply:
hmm từ này khoai quá nhỉ, mấy lần mình gặp cũng k biết dịch thế nào cho đúng

Reply:
mình thấy `bất thường` ổn mà, cũng có thể xài từ `đặc biệt`.

Reply:
Ở context này thì thực ra mình cũng k hiểu `nontrivial` có ý nghĩa là gì.
Mình mới chỉ hay nghe cụm `nontrivial solution` cho hệ phương trình.

Reply:
Nếu bí quá thì mình không bám theo bản gốc nữa được không?
```suggestion
Trong ví dụ bên dưới, ta sẽ tự tạo một phân phối để chứng minh luận điểm trên.
```

cc: @lkhphuc , @minhduc0711